src/components/lablup-loading-spinner.ts

```diff
@@ -33,22 +33,54 @@ export default class LablupLoadingSpinner extends LitElement {
     return [
       // language=CSS
       css`
-        wl-progress-spinner {
-          --progress-spinner-size: 48px;
-          --progress-spinner-stroke-width: 12px;
-          width: 48px;
-          height: 48px;
-          position: fixed;
-          bottom: 60px;
-          right: 60px;
+      .spinner-box {
+        width: 100px;
+        background-color: transparent;
+      }
+      .pulse-container {
+        width: 100px;
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+      }
+      .pulse-bubble {
+        width: 16px;
+        height: 16px;
+        border-radius: 50%;
+        background-color: rgba(114, 235, 81, 0.8);
+      }
+      .pulse-bubble-1 {
+          animation: pulse .4s ease 0s infinite alternate;
+      }
+      .pulse-bubble-2 {
+          animation: pulse .4s ease .2s infinite alternate;
+      }
+      .pulse-bubble-3 {
+          animation: pulse .4s ease .4s infinite alternate;
+      }
+      @keyframes pulse {
+        from {
+          opacity: 1;
+          transform: scale(1);
         }
+        to {
+          opacity: .25;
+          transform: scale(.75);
+        }
+      }
       `];
   }
 
   render() {
     // language=HTML
     return html`
-      <wl-progress-spinner id="spinner"></wl-progress-spinner>
+      <div class="spinner-box" id="dots">
+        <div class="pulse-container">  
+          <div class="pulse-bubble pulse-bubble-1"></div>
+          <div class="pulse-bubble pulse-bubble-2"></div>
+          <div class="pulse-bubble pulse-bubble-3"></div>
+        </div>
+      </div>
     `;
```

Comment:
I Cannot find any element that has `spinner` as its id.
And It seems like triggering "Cannot read the property of null" error due to[ this code](https://github.com/lablup/backend.ai-webui/pull/1088/files#diff-1dcfe4880af082ebb2cb8fc2c709fb67c57f3f85427fff91c827b76f00e58ca7R92).